package.json

```diff
@@ -16,8 +16,8 @@
     "nuke": "npm run clean && rm -rf node_modules/",
     "postinstall": "npm run build",
     "start": "babel-node server",
-    "test:integration": "NODE_ENV=development babel-node integration-tests/testcafeRunner.js",
-    "test:integration_win": "babel-node integration-tests/testcafeRunner.js",
+    "test:integration": "NODE_ENV=development babel-node testcafeRunner.js",
```

Comment:
any reason why we are moving the testcafeRunner into integration-tests folder?

Reply:
moving it out! it was the only thing in there

Reply:
👍

Reply:
ah sorry I got my reds and greens mixed up. What you've done is what I expected LOL